backend/src/main/java/com/bakdata/conquery/models/forms/managed/ManagedForm.java

```diff
@@ -136,16 +134,10 @@ public void start() {
 	@Override
 	public void addResult(@NonNull MasterMetaStorage storage, FormSharedResult result) {
 		ManagedExecutionId subquery = result.getSubqueryId();
-		if (subquery == null) {
-			fail(storage);
-			log.warn(
-				"Form failed in query plan creation. ",
-				new MultiException(result.getResults().stream()
-					.filter(r -> (r instanceof FailedEntityResult))
-					.map(FailedEntityResult.class::cast)
-					.map(FailedEntityResult::getThrowable)
-					.collect(Collectors.toList())));
-			return;
+		if(result.getError().isPresent()) {
+			fail(storage, result.getError().get());
+			log.warn("Form failed in query plan creation. {}", result.getError().get());
```

Comment:
Stimmt die log message?

Reply:
```suggestion
			log.warn("Form failed in query plan creation.", result.getError().get());
```

Reply:
Habe das log entfernt, da schon in finish geloggt wird.